hail/python/hail/context.py

```diff
@@ -141,7 +141,7 @@ def default_reference(self) -> ReferenceGenome:
         return self._default_ref
 
     @default_reference.setter
-    def set_default_reference(self, value):
+    def default_reference(self, value):
```

Comment:
Huh, I would not have expected this to work (two methods with the same name), but indeed it works.
```python3
In [1]: class Foo:
   ...:     def __init__(self):
   ...:         self._x = 3
   ...:     @property
   ...:     def x(self):
   ...:         return self._x
   ...:     @x.setter
   ...:     def x(self, new_x):
   ...:         self._x = new_x
   ...: 

In [2]: x = Foo()

In [3]: x.x = 3

In [4]: x.x
Out[4]: 3

In [5]: x.x = 10

In [6]: x.x
Out[6]: 10
```

Reply:
In fact, they have to have the same name. It doesn't work otherwise.